hs_restclient/__init__.py

```diff
@@ -1 +1,666 @@
-__author__ = 'miles'
+"""
+
+Client library for HydroShare REST API
+
+"""
+
+__title__ = 'hs_restclient'
+__version__ = '1.0.0'
+
+import os
+import datetime
+import zipfile
+import tempfile
+import shutil
+
+import requests
+
+from .compat import http_responses
+from .util import is_sequence
+
+
+STREAM_CHUNK_SIZE = 100 * 1024
+
+
+class HydroShareException(Exception):
+    def __init__(self, args):
+        super(HydroShareException, self).__init__(args)
+
+
+class HydroShareArgumentException(HydroShareException):
+    def __init__(self, args):
+        super(HydroShareArgumentException, self).__init__(args)
+
+
+class HydroShareNotAuthorized(HydroShareException):
+    def __init__(self, args):
+        super(HydroShareNotAuthorized, self).__init__(args)
+        self.method = args[0]
+        self.url = args[1]
+
+    def __str__(self):
+        msg = "Not authorized to perform {method} on {url}."
+        return msg.format(method=self.method, url=self.url)
+
+    def __unicode__(self):
+        return self.__str__()
```

Comment:
yes. Actually you could just return `unicode(str(self))` because while there is no `unicode()` in Python 3, it won't get called because `__unicode__` isn't called in Python 3.